app/assets/v2/js/pages/dashboard.js

```diff
@@ -74,6 +74,10 @@ function debounce(func, wait, immediate) {
   };
 }
 
+var scrub = function(v) {
```

Comment:
thought on updating it to `const scrub = value => value.replace(/[\W]+/g, '')` ?